apps/site/lib/site_web/templates/alert/_banner.html.eex

```diff
@@ -1,18 +1,18 @@
-<% label = Alerts.Alert.human_label(@alert) %>
+<% label = BannerAlert.human_label(@alert) %>
 <div tabindex="0" class="c-alert-item c-alert-item--system">
   <div class="c-alert-item__icon">
-    <%= @alert |> Alerts.Alert.icon() |> alert_icon() %>
+    <%= @alert |> BannerAlert.icon() |> alert_icon() %>
   </div>
   <div class="c-alert-item__top">
     <div class="c-alert-item__top-text-container">
       <div class="c-alert-item__effect">
-        <%= Alerts.Alert.human_effect(@alert) %>
+        <%= BannerAlert.human_effect(@alert) %>
         <%= unless label == "" do
-          content_tag(:span, [label], class: alert_label_class(@alert))
+          content_tag(:span, [label], class: @alert |> BannerAlert.alert_for_label_class() |> alert_label_class)
```

Comment:
Can you make `label_class` a part of the `BannerAlert` interface? `BannerAlert.label_class(@alert)`